.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,152 @@
+
+"""
+# standard imports
+import argparse
+import json
+import os
+import sys
+from collections import namedtuple
+
+# Named tuple for file and missing translations combination
+FileTranslation = namedtuple(
+    "FileTranslation", ["file", "missing_translations"]
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)